pynest/nest/tests/test_quantal_stp_synapse.py

```diff
@@ -101,15 +97,16 @@ def test_QuantalSTPSynapse(self):
         vm_reference = numpy.array(nest.GetStatus(
             [voltmeter[0]], 'events')[0]['V_m'])
 
-        vm.shape = (n_trials, t_tot)
-        vm_reference.shape = (n_trials, t_tot)
+        vm.shape = (n_trials, int(t_tot))
+        vm_reference.shape = (n_trials, int(t_tot))
```

Comment:
Using `int(t_tot)` here to dimension the result array is not a good idea, since you implicitly require that the voltmeter records at 1 ms intervals. A cleaner implementation would be (not 100% sure of spelling of assertions)
```python
self.assert_equal(len(vm) % n_trials, 0)
n_steps = int(len(vm) / n_trials)    # int to be absolutely Py3-safe
vm.shape = (n_trials, n_steps)
...
```